apps/server/src/services/integration-service/OscIntegration.ts

```diff
@@ -113,6 +110,44 @@ export class OscIntegration implements IIntegration<OscSubscription, OSCSettings
     }
   }
 
+  private stringToOSCArgs(argsString): ArgumentType[] {
+
+    // NOTE: regex taken from https://stackoverflow.com/questions/4031900/split-a-string-by-whitespace-keeping-quoted-segments-allowing-escaped-quotes
+    const parseRegex = /[\w.]+|"(?:\\"|[^"])+"/g;
+
+    const parsedArguments = argsString.match(parseRegex).map((argString: string) => {
+      const argAsNum = Number(argString);
+
+      // NOTE: number like: 1 2.0 33333
+      if (!Number.isNaN(argAsNum)) {
+        if (argString.includes('.')) {
+          return {
+            type: 'f',
+            value: argAsNum,
+          };
+        } else {
+          return {
+            type: 'i',
+            value: argAsNum,
+          };
+        }
+      }
+
+      if (argString.startsWith('"') && argString.endsWith('"')) {
+        // NOTE: "quoted string"
+        return {
+          type: 's',
+          value: argString.substring(1, argString.length - 1).replaceAll('\\"', '"'),
+        };
+      }
+      return {
+        type: 's',
+        value: argString,
+      };
+    });
```

Comment:
suggestion: similar possible improvement here, since we are only changing the value we could make that change explicit.
Something like (untested):

```ts
  const isQuotedString = argString.startsWith('"') && argString.endsWith('"');
  const value = isQuotedString ? argString.substring(1, argString.length - 1).replaceAll('\\"', '"') : argString;
  return { type: 's', value };
```